opendistro-elasticsearch-security.release-notes

```diff
@@ -1,4 +1,9 @@
-## 2019-11-18 Version 1.3.0.0 (Current)
+## 2020-01-15 Version 1.4.0.0
+
+- Support for Elasticsearch 7.4.2
+- Removing support of OpenSSL for Java 12+
+
```

Comment:
also add "Merging of all security repos (security-parent and security-advanced-modules in security.) "

Reply:
Added.